src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -891,7 +891,15 @@ PerfAnalyzer::Run(int argc, char** argv)
         break;
       case 5: {
         std::string arg = optarg;
-        std::string name = arg.substr(0, arg.rfind(":"));
+        auto colon_pos = arg.rfind(":");
+        if (colon_pos == std::string::npos) {
+          Usage(
+              argv,
+              "failed to parse input shape. There must a colon after input "
+              "name.");
+          break;
```

Comment:
no need for this, `Usage()` calls `exit()` at the end

Reply:
Fixed.